.dotnet/src/Generated/Models/ChatCompletion.Serialization.cs

```diff
@@ -34,6 +34,11 @@ void IJsonModel<ChatCompletion>.Write(Utf8JsonWriter writer, ModelReaderWriterOp
             writer.WriteNumberValue(CreatedAt, "U");
             writer.WritePropertyName("model"u8);
             writer.WriteStringValue(Model);
+            if (Optional.IsDefined(LatencyTierOutcome))
+            {
+                writer.WritePropertyName("service_tier"u8);
+                writer.WriteStringValue(LatencyTierOutcome.Value.ToString());
+            }
```

Comment:
According to the REST API, this is nullable. I think this means that this code is a little different and uses `writer.WriteNull`? Would be good to double-check: Does it look different if you delete the custom serialization logic and regenerate?

The deserialize method might also look a little different. I think it explicitly assigns `null`? Good to double-check both.